modules/custom/openy_upgrade_tool/config/install/openy_upgrade_tool.settings.yml

```diff
@@ -0,0 +1 @@
+force_mode: 0
```

Comment:
1 by default
We should allow user to go to the end no matter what. Otherwise upgrade is time consuming

Reply:
Fixed.